projects/map-ol/src/lib/map-ol.service.ts

```diff
@@ -1660,7 +1673,12 @@ export class MapOlService {
 
       let overlayoptions = defaultOptions;
 
-      if (this.isPopupObj(layerpopup) && layerpopup.options) {
+      if (this.isPopupObjArray(layerpopup)) {
```

Comment:
This is the most important change here, right?
Honestly, I don't think it's trivially obvious what is happening here. Could you add a comment line?

Reply:
Added a comment https://github.com/dlr-eoc/ukis-frontend-libraries/commit/5124af262cb84692b84fd248e73578a1faef376b